src/Components/Nav/Nav.css

```diff
@@ -1,5 +1,5 @@
 .nav-menu__links.navbar-collapse {
```

Comment:
I QA this and it doesn't look or function in mobile as we talked: https://getbootstrap.com/docs/4.2/examples/starter-template/

Also, the menu doesn't work in the iPhone (not responsibe) the device it self.